assets/scripts/util/street_analytics.js

```diff
@@ -61,46 +63,111 @@ const CAPACITIES = {
   'magic-carpet': { average: 2, potential: 3 }
 }
 
-const hasCapacityType = (type) => {
+const CAPACITIES_FOO = {
+  sidewalk: { average: 1500, potential: 1500 },
+  'drive-lane': { average: 1500, potential: 1500 },
+  'bike-lane': { average: 1500, potential: 1500 },
+  scooter: { average: 1500, potential: 1500 },
+  'light-rail': { average: 1500, potential: 1500 },
+  streetcar: { average: 18000, potential: 1500 },
+  'bus-lane': { average: 5000, potential: 8000 },
+  'magic-carpet': { average: 1500, potential: 1500 }
+}
+
+const CAPACITIES_BAR = {
+  sidewalk: { average: 5000, potential: 5000 },
+  'drive-lane': { average: 5000, potential: 5000 },
+  'bike-lane': { average: 5000, potential: 5000 },
+  scooter: { average: 5000, potential: 5000 },
+  'light-rail': { average: 18000, potential: 18000 },
+  streetcar: { average: 18000, potential: 20000 },
+  'bus-lane': { average: 5000, potential: 8000 },
+  'magic-carpet': { average: 2, potential: 3 }
+}
+
+const hasCapacityType = (type, source = '') => {
+  if (source === 'foo') {
+    return type in CAPACITIES_FOO
+  }
+  if (source === 'bar') {
+    return type in CAPACITIES_BAR
+  }
   return type in CAPACITIES
 }
 
 export const getCapacity = (type) => {
   return hasCapacityType(type) ? { ...CAPACITIES[type] } : UNDEFINED_CAPACITY
 }
 
+export const getCapacityBySource = (type, source) => {
+  if (source === '' || source === 'defaultSource') {
+    return hasCapacityType(type) ? { ...CAPACITIES[type] } : UNDEFINED_CAPACITY
+  }
+
+  if (source === 'foo') {
+    return hasCapacityType(type, source)
+      ? { ...CAPACITIES_FOO[type] }
+      : UNDEFINED_CAPACITY
+  }
+
+  if (source === 'bar') {
+    return hasCapacityType(type, source)
+      ? { ...CAPACITIES_BAR[type] }
+      : UNDEFINED_CAPACITY
+  }
+
+  return hasCapacityType(type) ? { ...CAPACITIES[type] } : UNDEFINED_CAPACITY
+}
```

Comment:
Similarly, let's find a way to refer to data without having to hard-code logic and references to each.